File: Java   Assignments/Dumebi Week3 Assignment2/gradingSystem.java
import java.util.Scanner;

public class gradingSystem {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);
        int i = 20;
        while (i < 100) {
            System.out.print("Enter your grade: ");
            int grade_1 = input.nextInt();

            System.out.print("Enter your grade: ");
            int grade_2 = input.nextInt();

            System.out.print("Enter your grade: ");
            int grade_3 = input.nextInt();

            double average_grade = (grade_1 + grade_2 + grade_3) / 3.0;
            System.out.println(average_grade);

            if (90 <= average_grade && average_grade <= 100) {
                System.out.println("A");
            } else if (80 <= average_grade && average_grade < 90) {
                System.out.println("B");
            } else if (70 <= average_grade && average_grade < 80) {
                System.out.println("C");
            } else if (60 <= average_grade && average_grade < 70) {
                System.out.println("D");
            } else if (0 <= average_grade && average_grade < 60) {
                System.out.println("F");
            }
        }
    }
}

File: Python Assignments/Week 3 Assignment/grading_system.py
# Number 2

grade_1 = int(input("Enter your grade: "))
grade_2 = int(input("Enter your grade: "))
grade_3 = int(input("Enter your grade: "))

average_grade = (grade_1 + grade_2 + grade_3) / 3
print(average_grade)

if 90 <= average_grade <= 100:
     print("A")
elif 80 <= average_grade < 90:
     print("B")
elif 70 <= average_grade < 80:
    print("C")
elif 60 <= average_grade < 70:
    print("D")
elif 0 <= average_grade < 60:
    print("F")